zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterSettingManager.java

```diff
@@ -245,26 +245,11 @@ private void loadFromFile() throws IOException {
 
       InterpreterSetting interpreterSettingTemplate =
           interpreterSettingTemplates.get(savedInterpreterSetting.getGroup());
-      // InterpreterSettingTemplate is from interpreter-setting.json which represent the latest
+      // InterpreterSettingTemplate is from interpreter-setting.json which represent the initialized
       // InterpreterSetting, while InterpreterSetting is from interpreter.json which represent
       // the user saved interpreter setting
       if (interpreterSettingTemplate != null) {
-        savedInterpreterSetting.setInterpreterDir(interpreterSettingTemplate.getInterpreterDir());
```

Comment:
```
savedInterpreterSetting.setInterpreterDir(interpreterSettingTemplate.getInterpreterDir());
```
This `setInterpreterDir()` property is not copied in `InterpreterSetting.java` above.
Can this line of code be deleted?

Reply:
Correct, I should keep it. will fix it